nest/iface.c

```diff
@@ -838,3 +845,14 @@ if_show_summary(void)
     }
   cli_msg(0, "");
 }
+
+static int
+if_delete_request(struct iface *i) {
+    int del = 0;
+
+    if ((i->flags - IF_SHUTDOWN == 0U) && !i->index && !i->mtu && !i->addr &&
```

Comment:
Could you give a bit more detail on why we need to check the index / mtu / addr as well as the flags?

Is checking whether or not the interface was shutdown not enough?

Reply:
Also, I think you probably want `(i->flags & IF_SHUTDOWN)` for the flag check, right?